### ISSUE TITLE
Feature request: allow changing the windows controls position from right to left

### ISSUE THREAD

**Issue by vadrian89** [linuxdeepin/dde-kwin#135]
Hello, 

Since Deepin switched to Kwin is it possible to introduce a feature to allow users to change window controls position from right to the left?

**Comment by xmuli** [comment]
Shift + TAB ？

**Comment by vadrian89** [comment]
No. I mean, to move the Close, Minimize and Maximize buttons to the left.

**Comment by xmuli** [comment]
Then this requires DDE's product to consider this design

**Comment by vadrian89** [comment]
If this isn't the place for this request, could you please point me to the correct repository? 
dde-wm is not available and file-manager doesn't seem the apropriate place.
